.github/workflows/ci.yml

```diff
@@ -27,9 +27,9 @@ jobs:
           python -m pip install --disable-pip-version-check .
       - name: Run tests on ${{ matrix.os }}
         run: nox --non-interactive --error-on-missing-interpreter --session "tests-${{ matrix.python-version }}" -- --full-trace
-        if: matrix.python-version != '3.11-dev'
+        if: matrix.python-version != '3.11'
       - name: Run tests on ${{ matrix.os }}
-        if: matrix.python-version == '3.11-dev'
+        if: matrix.python-version == '3.11'
         run: nox --non-interactive --error-on-missing-interpreter --session "tests-3.11" -- --full-trace
```

Comment:
We can just get rid of the `if` entirely now can't we and just use the `{{ matrix.python-version }}`?

Reply:
Ahh, yes, I think so. Can try to do it tonight.